docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -109,8 +109,8 @@ Drop the ``--user`` option in the following commands if you use a virtual enviro
 
    .. code-block:: bash
 
-    pip install --user -r docs/docsite/requirements.txt # This file installs the unpinned dependency versions.
-    pip install --user -r tests/requirements.txt # This file installs tested dependency versions.
+    pip install --user -r tests/requirements.in -c tests/requirements.txt # Installs tested dependency versions.
```

Comment:
In a follow up, let's get rid of `--user` mentions in the docs and show setting up a venv. `--user` doesn't even work on new Debian versions. It's bad practice. This is especially problematic with constraints files that can really mess up your global environment.

Reply:
@gotmax23 I think that one of the downsides of `--user` is that it shares `bin/` across Python interpreters of different versions, causing collisions.